### PR TITLE
updating webhook pipeline docs and minor enhancements

### PR DESCRIPTION
**Pull request by renzokuken**
Made improvements detailing specific components of the webhook pipeline and how to test. Re-pointed dataflow template to teleport. Added metadata enrichment for alooma backwards compatibility.

### CODE REVIEW

File: tools/webhook-ingestion-pipeline/webhook_endpoint/main.py [hunk 1]
```diff
@@ -51,6 +55,22 @@ def handle_invalid_usage(error):
...
+    """ Appends necessary _metadata fields s
+        for downstream processing
+        and event splitting
+    """
+    default_metadata = {'@uuid': str(uuid.uuid4()),
+                        '@timestamp': datetime.utcnow().isoformat()}
+    if isinstance(json_data, str):
+        json_data = json.loads(json_data)
+
+    if '_metadata' not in json_data.keys():
```
Comment (jaketf) [inline tools/webhook-ingestion-pipeline/webhook_endpoint/main.py:69]: unecessary call to `dict.keys`.
```suggestion
    if '_metadata' not in json_data:
```

File: tools/webhook-ingestion-pipeline/webhook_endpoint/main.py [hunk 2]
```diff
@@ -51,6 +55,22 @@ def handle_invalid_usage(error):
     response.status_code = error.status_code
     return response
 
+
+def _append_metadata(json_data):
```
Comment (jaketf) [inline tools/webhook-ingestion-pipeline/webhook_endpoint/main.py:59]: nit: this is not 'append' really it's falling back to default metadata.
perhaps use `_set_missing_metadata`

File: tools/webhook-ingestion-pipeline/webhook_endpoint/main.py [hunk 3]
```diff
@@ -51,6 +55,22 @@ def handle_invalid_usage(error):
...
+    """ Appends necessary _metadata fields s
```
Comment (jaketf) [inline tools/webhook-ingestion-pipeline/webhook_endpoint/main.py:60]:
```suggestion
    """ Appends necessary _metadata fields
```

**Review by jaketf — APPROVED** [review]
just a few nits.
@dhercher PTAL docs.

**Review by dhercher — APPROVED** [review]

**Pull request merged**